pkg/fake/cloudprovider.go

```diff
@@ -71,6 +71,10 @@ func (c *CloudProvider) Get(context.Context, string) (*v1alpha5.Machine, error)
 	return nil, nil
 }
 
+func (c *CloudProvider) List(context.Context) ([]*v1alpha5.Machine, error) {
```

Comment:
Why is this being added?

Reply:
It's conforming to the changes that were added to the cloudprovider interface in [this PR](https://github.com/aws/karpenter-core/pull/249)